brain_games/games/utils.py

```diff
@@ -0,0 +1,11 @@
+from random import randint
+from typing import Dict
+
+
+def get_random_number(bottom: int, ceiling: int) -> int:
+    return randint(bottom, ceiling)
+
+
+def generate_game_round_data(d1: str, d2: str, d3: str) -> Dict[str, str]:
```

Comment:
d1, d2, d3 are bad argument names. what d1 would say to anyone who reads this code? it's better to replace them with rule, question, correct_answer